R/import.R

```diff
@@ -63,6 +63,9 @@
 #'   factorial sampling design.
 #' @param hierarchy a hierarchical formula that explicitely defines hierarchical
 #'   levels in your strata. see \code{\link{hierarchy}} for details.
+#' @param check.ploidy a boolean indicating if the ploidy should be checked or
+#' is assumed to be correct. Skipping this step improves significantly the time
+#' needed to create a genind object. 
```

Comment:
A couple of things:

1. It would be good to indicate the default setting (`TRUE`) and explain what it does.
2. Add a sentence to gently warn the user to test importing a subset of their data before turning this option off as silent bugs have occurred in the past (i.e. #216)